infrastructure/src/main/java/org/corfudb/infrastructure/log/statetransfer/batchprocessor/BatchProcessorFailure.java

```diff
@@ -0,0 +1,22 @@
+package org.corfudb.infrastructure.log.statetransfer.batchprocessor;
+
+
+import lombok.Getter;
+import org.corfudb.infrastructure.log.statetransfer.StateTransferException;
+
+/**
+ * A state transfer exception that is propagated to the caller of a batch processor,
+ * after a batch processor failed to handle a batch transfer.
+ */
+@Getter
+public class BatchProcessorFailure extends StateTransferException {
```

Comment:
please rename to `BatchProcessorException`. I agree BatchProcessorFailure sounds pretty good, but it is hard to find, because a developer always knows that any exception class has Exception in its suffix

Reply:
Was removed.